examples/sdk-examples/src/examples/chartResponsiveness/index.tsx

```diff
@@ -28,12 +37,25 @@ export const ChartResponsiveness: React.FC = () => (
         <h2 id="bar-chart">Visual Components</h2>
         <p>
             Small visual components will have parts of the chart hiden to improve user responsiveness
-            experience.
+            experience. Various widget heights can be seen in this example. Smallest has only the axis title
+            visible, middle has only the axis labels visible and the largest has both axis title and labels
+            visible.
         </p>
         <ExampleWithSource
             for={BarChartResponsiveExample}
             source={BarChartResponsiveExampleSrc}
             sourceJS={BarChartResponsiveExampleSrcJS}
         />
+        <h2>Visual Components: responsive legend</h2>
+        <p>
+            Demonstration of legend responsive behaviour. Try changing legend responsive config for different
+            chart dimensions. When configured, it can be seen that the legend has forced position and pop-up
+            functionality for smaller dimensions.
```

Comment:
<h2>Responsive Legend</h2>
<p>To see a responsive legend, try various combinations of the legend position/responsiveness and the container dimensions.
When the responsiveness is set to "autoPositionWithPopup", you can see that the legend has a forced position and appears as a popup in smaller containers.